docs/_pages/0002-maintainers.md

```diff
@@ -11,9 +11,9 @@
 
 title: Maintainers
 layout: post
-next_url: /docs/navigation/
-prev_url: /
+prev_url: /docs/contact/
 permalink: /docs/maintainers/
+next_url: /docs/changelog/
```

Comment:
I have mixed feelings; both going to `/docs/navigation/` and going to `/docs/changelog` have merit:

One of them means that this is a non-critical section and we are jumping to the meat of the content.

And the other would mean we are following things in sequential order.

I think it’s okay to keep it as `/docs/changelog` though. The user can skip a few pages further by pressing "next" if they are not interested in the content anyway.

Reply:
I agree to the point of skipping Changelog and Roadmap for someone who's new taking look at the project.
But then this might look like a bug if it doesn't go to next page in sequence.
Maybe we can think of placing Changelog and Roadmap to another location, where it's easier for needy to go.

Reply:
Leaving it to you @v0lkan to decide on above, we can create an issue if we want to do above.

Reply:
Suggestion 💡: What do you think keeping Changelog and Roadmap along with Release Management?
So it doesn't get in the way for a new onlooker.